src/components/Inputs/TextInput/TextInput.tsx

```diff
@@ -14,6 +16,7 @@ export const TextInput: FC<IProps & TestingProps> = ({
   disabled = false,
   placeholder,
   type,
+  helper,
```

Comment:
Nitpick : I find `helper` confusing as it can be interpreted in many ways, notably a `helper` function or some dependency.

I suggest `infoText` or something similar :)

Reply:
`description`, `tooltipContent`?